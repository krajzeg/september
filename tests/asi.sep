print("Omitting semicolons should be OK.")
if (2 < 3) { print("Also inside braces.") }
if (3 < 4) {
	print("But it should not break multi-line blocks.")
}

print("Now, let's try an operator on the next line.")
upper_version := locals.version
    .upper()
print(upper_version)
