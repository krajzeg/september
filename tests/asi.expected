Omitting semicolons should be OK.
Also inside braces.
But it should not break multi-line blocks.
Now, let's try an operator on the next line.
0.1-APPLES
